packages/nx/src/executors/utils/convert-nx-executor.ts

```diff
@@ -39,7 +33,7 @@ export function convertNxExecutor(executor: Executor) {
         nxJsonConfiguration,
         workspace: { ...projectsConfigurations, ...nxJsonConfiguration },
         cwd: process.cwd(),
-        projectGraph,
+        projectGraph: null,
```

Comment:
@vsavkin do you think it'd be worth doing something like this to be less breaking?

```
projectGraph: builderContext.workspaceRoot === workspaceRoot ? projectGraph : null
```